gwt/src/main/resources/org/pentaho/ui/database/resources/databasedialog.xul

```diff
@@ -63,7 +63,7 @@
 						</listbox>
             <hbox>
               <label flex="1" id="context_help" value="${DatabaseDialog.label.AddingDatabases}" onclick="dataHandler.showContextHelp()"/>
-              <image src="help_link.png" height="16" width="16" onclick="dataHandler.showContextHelp()"/>
+              <image src="help_link_grayscale.png" height="16" width="16" onclick="dataHandler.showContextHelp()"/>
```

Comment:
@pentaho-amessier This change will use the grayscale help link icon across all themes, not just Ruby.

Reply:
Thanks - I missed the comment in JIRA about the ruby theme.